storage.bs

```diff
@@ -229,6 +229,17 @@ settings object</a> <var>environment</var>, run these steps:
  <li><p>Return <var>key</var>.
 </ol>
 
+<p>To determine whether a <a>storage key</a> <var>A</var>
+<dfn export for="storage key" id=concept-storage-key-equals lt=equal>equals</dfn> <var>B</var>, run
```

Comment:
Let's repeat "a storage key" in front of _B_ for maximum clarity.

Reply:
Also no need to include an ID here. For new things we let Bikeshed generate one.

Reply:
I modeled it off of this: https://github.com/whatwg/url/blob/main/url.bs#L1144-L1146

There they didn't repeat the type for `B`, but I agree that it's helpful for maximum clarity. Also, manually adding the id lets us prepend `concept` to it, which seems like something that's done in a lot of places (without the id specified, bikeshed makes it `storage-key-equals`). I'm happy to drop it, though.

Reply:
Yeah, `concept-` is a legacy pre-Bikeshed thing.

Also, always fun to have your own code come back to bite you. I can fix that one.

Reply:
https://github.com/whatwg/url/pull/686